include/util/hex_float.h

```diff
@@ -39,6 +39,16 @@
 
 namespace spvutils {
 
+class Float16 {
+ public:
+  Float16(uint16_t v): val(v) {}
+  Float16() {}
```

Comment:
The default constructor will not get created if we specify a constructor, so in order to have one, we have to write it.